pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -540,7 +574,7 @@ func buildDefaultCluster(env *model.Environment, name string, discoveryType v2.C
 	}
 
 	defaultTrafficPolicy := buildDefaultTrafficPolicy(env, discoveryType)
-	applyTrafficPolicy(cluster, defaultTrafficPolicy, nil)
+	applyTrafficPolicy(cluster, defaultTrafficPolicy, nil, env.Mesh.SdsRefreshDelay, env.Mesh.SdsUdsPath)
```

Comment:
Isn't SDS push based ? Why does it have a refresh delay ?

Reply:
removed ```sdsrefreshdelay``` in https://github.com/istio/istio/pull/8843 and https://github.com/istio/api/pull/637